platform_config.go

```diff
@@ -6,56 +6,25 @@ import (
 	"net/http"
 	textTemplate "text/template"
 
+	_ "embed"
+
 	"github.com/gofrs/uuid"
 	"github.com/gorilla/mux"
 	"github.com/rs/zerolog/log"
 )
 
+//go:embed static/apple.html
```

Comment:
can you call this folder `templates/` instead of static?

Reply:
sure